corehq/apps/groups/models.py

```diff
@@ -148,11 +159,7 @@ def get_static_users(self, is_active=True):
 
     @classmethod
     def by_domain(cls, domain):
```

Comment:
I think if we're able to robustly refresh the views whenever changes are made, it's fine to leave the name as is